docker-entrypoint.sh

```diff
@@ -113,6 +113,14 @@ if [[ "$EXTENSIONS_REPLICATION_DEST_BOOTSTRAPLIST" ]]; then
     fi
 fi
 
+if [[ "$EXTENSIONS_REPLICATION_REPLICATION_STATUS_PROCESSOR_RETRY_TIMEOUT_S" ]]; then
```

Comment:
should we just reduce the name to just use "replication" once?

Reply:
They have separate configs at the moment, so I introduced both. And honestly, there is no need to change replication status processor retries. It's the QueueProcessor that counts as it affects the backoff time for retries to cloud locations.